askbot/media/js/post.js

```diff
@@ -2199,7 +2206,7 @@ EditCommentForm.prototype.getSuppressEmail = function () {
 };
 
 EditCommentForm.prototype.setSuppressEmail = function (bool) {
-    this._element.find('input[name="suppress_email"]').prop('checked', bool);
+    this._element.find('input[name="suppress_email"]').prop('checked', bool).trigger('change');
```

Comment:
ok. It worked because the property is read just before making the submit.
Nothing was listening to the change event.